src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  letter-spacing: 0;
+}
+
+.header {
+  min-height: 60px;
+  display: flex;
```

Comment:
Please pay attention to your CSS properties order
https://9elements.com/css-rule-order/